tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -278,11 +280,14 @@ tensorflow::Status ConvertSubGraphToTensorRT(ConvertGraphParams* params) {
     subgraph_edge_to_output_map.insert({params->subgraph_outputs.at(i), i});
   }
   TF_RETURN_IF_ERROR(status);
+  unique_tensors.clear();
```

Comment:
This is not used in the following, do we need to do similar things to the outputs? I guess not as there shouldn't be duplicates on the output (not like the input). If not, please remove this line.

Reply:
Output also has the same issue but later decided to handle it elsewhere. Line is removed